test/NuGet.Core.FuncTests/Dotnet.Integration.Test/MsbuildIntegrationTestFixture.cs

```diff
@@ -405,12 +415,28 @@ private static void DeleteDirectory(string path)
             }
             catch (UnauthorizedAccessException)
             {
-                Directory.Delete(path, true);
+                // Ignore UnauthorizedAccessException for the first attempts
+                var MaxTries = 100;
+
+                for (var i = 0; i < MaxTries; i++)
+                {
+                    
+                    try
+                    {
+                        Directory.Delete(path, true);
```

Comment:
Add named parameter for the pesky literal `true`.

Reply:
It was copied from Heng's branch so I didn't want merge conflicts, but I can fix it up :)